packages/ui/src/memberships/components/MemberProfile/MemberRoles.tsx

```diff
@@ -0,0 +1,137 @@
+import { Member } from '../../types'
+
+export const MemberSideRoles = ({ member }: { member: Member }) => {
+  return (
+    <RolesDisplay gap={8}>
+      {member.roles ? <MemberSideRole member={member} /> : <TextMedium light>This profile has no roles.</TextMedium>}
+    </RolesDisplay>
+  )
+}
+
+const MemberSideRole = ({ member }: { member: Member }) => {
+  return (
+    <MemberRoleToggle absoluteToggle>
+      {(isOpen) => {
+        return (
+          <MemberRoleToggleContainer>
+            <MemberRoleTitleContainer>
+              <h5>Name of role</h5>
+            </MemberRoleTitleContainer>
+            {isOpen && (
+              <MemberRoleTableContainer>
+                <MemberRoleTable>
+                  <SidePaneRow>
+                    <SidePaneLabel text="Hired" />
+                    <SidePaneColumn>Add block component here</SidePaneColumn>
+                  </SidePaneRow>
+                  <SidePaneRow>
+                    <SidePaneLabel text="Reward" />
+                    <SidePaneColumn>
+                      <SidePaneText>13 923 JOY / 3600 blocks</SidePaneText>
+                    </SidePaneColumn>
+                  </SidePaneRow>
+                  <SidePaneRow>
+                    <SidePaneLabel text="Earned total" />
+                    <SidePaneColumn>
+                      <SidePaneText>20 923M JOY </SidePaneText>
+                    </SidePaneColumn>
+                  </SidePaneRow>
+                  <SidePaneRow>
+                    <SidePaneLabel text="Earned in 24h" />
+                    <SidePaneColumn>
+                      <SidePaneText> {member.id} JOY (id for test) </SidePaneText>
+                    </SidePaneColumn>
+                  </SidePaneRow>
+                  <SidePaneRow>
+                    <SidePaneLabel text="Role account" />
+                    <SidePaneColumn>Add account component here</SidePaneColumn>
+                  </SidePaneRow>
+                  <SidePaneRow>
+                    <SidePaneLabel text="Staking account" />
+                    <SidePaneColumn>Add account component here</SidePaneColumn>
+                  </SidePaneRow>
+                  <SidePaneRow>
+                    <SidePaneLabel text="Reward account" />
+                    <SidePaneColumn>Add account component here</SidePaneColumn>
+                  </SidePaneRow>
+                  <ButtonsGroup align="left">
+                    <ButtonGhost size="small">
+                      Application preview <Arrow direction="right" />
+                    </ButtonGhost>
+                    <ButtonGhost size="small">
+                      Opening preview <Arrow direction="right" />
+                    </ButtonGhost>
+                  </ButtonsGroup>
+                </MemberRoleTable>
+              </MemberRoleTableContainer>
+            )}
+          </MemberRoleToggleContainer>
+        )
+      }}
+    </MemberRoleToggle>
+  )
+}
```

Comment:
Could you extract this into a separate file and create a storybook story for it?

We can use that later to validate if the data we pass there is OK when we will implement member roles.